libs/service/rabbitmq/src/getAMQClient.ts

```diff
@@ -0,0 +1,10 @@
+import { AMQPClient } from "@cloudamqp/amqp-client";
+
+let client: AMQPClient;
+
+export const getAMQClient = async (serverUri: string): Promise<AMQPClient> => {
+	if (client) return client;
```

Comment:
nitpick, should be `getAMQPClient` but I'd prefer `getRabbitClient` and `getMongoClient`

Reply:
fair point